tableau_utilities/scripts/server_info.py

```diff
@@ -27,22 +27,22 @@ def server_info(args, server):
         sorted_records = sorted(object_list, key=lambda d: d[sort_field])
         if format == 'names':
             for record in sorted_records:
-                print(f'  {symbol.arrow_r}', f"{color.fg_yellow}{record['name']}{color.reset}")
+                print(record['name'])
```

Comment:
BTW, I think part of the issue I was having with the stylized printing was because I am using the 1password cli.  It appears that when I use that the stylized printing isn't really work.

In testing the connection function I wrote raw arguments to see if something with 1password was messing it up. Look at this difference in output

<img width="1698" alt="image" src="https://user-images.githubusercontent.com/9727812/225927292-fb0d47d4-3b13-4442-81b9-f455b58f7beb.png">